integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/generator.py

```diff
@@ -142,7 +142,8 @@ def __init__(
         if max_length is not None or truncate is not None:
             warnings.warn(
                 "The 'max_length' and 'truncate' parameters have been removed and no longer have any effect. "
```

Comment:
When are we planning to fully remove them from the init method if they are deprecated?